src/reader/reader.js

```diff
@@ -17,6 +17,18 @@ class Reader extends Component {
     this.state = {ready: false}
   }
 
+  componentDidMount() {
+    const ua = window.navigator.userAgent
+    if (ua.indexOf("iPhone") > -1 || ua.indexOf("iPad") > -1) {
```

Comment:
The iPad by default returns a Mac OS X user agent aka it always requests the Desktop site by default. The quick fix for the user is to request the mobile site instead of the desktop site to return the `iPad` specific user agent.